### PR TITLE
presenterm: 0.7.0 -> 0.8.0

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

Changelog: https://github.com/mfontanini/presenterm/releases/tag/v0.8.0

cc @mikaelfangel

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 330868` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>presenterm</li>
  </ul>
</details>

File: pkgs/by-name/pr/presenterm/package.nix [hunk 1]
```diff
@@ -2,46 +2,42 @@
...
   buildInputs = [
     libsixel
   ];
 
-  cargoHash = "sha256-w1uXCH8Ybf78EPTIKrhPlPHAnNBp1iiBpFJHY98IPWY=";
+  cargoHash = "sha256-2aHJnGSuP0TEBMxF1zljbEyk1g6ECTpnByyH8jaj78s=";
 
-  # Crashes at runtime on darwin with:
-  # Library not loaded: .../out/lib/libsixel.1.dylib
-  buildFeatures = lib.optionals (!stdenv.isDarwin) [ "sixel" ];
```
Comment (SuperSandro2000) [inline pkgs/by-name/pr/presenterm/package.nix:29]: Why not keep that?

Comment (GaetanLepage) [reply pkgs/by-name/pr/presenterm/package.nix:29]: I was not able to reproduce the bug. Do you happen to own a mac ?

Comment (GaetanLepage) [reply pkgs/by-name/pr/presenterm/package.nix:29]: I was able to present a slideshow with images on the darwin builder.
Images appear very blurred but I have not experimented any crash.

Comment (matteo-pacini) [reply pkgs/by-name/pr/presenterm/package.nix:29]: @GaetanLepage works on `aarch64-darwin` in WeTerm, no crashes 
image

Comment (mattpolzin) [reply pkgs/by-name/pr/presenterm/package.nix:29]: The removed code added sixel if not on a Mac -- so the removal of that code means non-darwin machines won't have sixel image support, not that Darwin will crash.

Comment (GaetanLepage) [reply pkgs/by-name/pr/presenterm/package.nix:29]: > I was able to present a slideshow with images on the darwin builder.
> Images appear very blurred but I have not experimented any crash.

This attempt was using `foot`.

Comment (mattpolzin) [reply pkgs/by-name/pr/presenterm/package.nix:29]: I guess my statement still stands, I just don’t know of a terminal that only supports sixel off hand.

Comment (mattpolzin) [reply pkgs/by-name/pr/presenterm/package.nix:29]: All I'm trying to say but might not be saying it clearly is that originally `sixel` was turned on for all builds; I turned it off conditionally for Darwin (https://github.com/NixOS/nixpkgs/commit/e89f61f917a046375990f56a5f6a713314dd8098) to fix a crash. So undoing my change would mean unconditionally building with sixel support, not deleting the code entirely.

If everyone agrees we should never build with sixel support, that's one thing, but the bug I fixed was related to always building with sixel, not never building with it.

Comment (GaetanLepage) [reply pkgs/by-name/pr/presenterm/package.nix:29]: Oh right, I completely misread the code, my attempt was to try to re-enable sixel for all platforms (which is not what I have done).

Comment (GaetanLepage) [reply pkgs/by-name/pr/presenterm/package.nix:29]: Nevermind, it still crashes. I have reverted the patch.

**Review by mattpolzin — APPROVED** [review]
Built & tested on `aarch64-darwin`.

**Pull request merged**